pages/docs/tutorials/create-asyncapi-document.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Create AsyncAPI Document
+description: In this tutorial, you'll learn how to create an AsyncAPI document.
+weight: 120
+---
+
+# Introduction
+
+In this tutorial, you'll learn how to create an AsyncAPI document based on a sample real-world use case. Additionally, you will learn about event-driven architecture, message brokers, pub/sub pattern.
+
+Let's pretend you have a company called Smarty Lighting, and you install smart-city streetlight lighting systems. This smart lighting system is a use case of the Internet of things (IoT).You will create a Smartylighting Streetlights API using Node.js and Mosquitto (MQTT) as the message broker. This API will allow you to manage city lights remotely. 
+
+You want to build a system that can turn streetlights on and off based on their environmental conditions: 
+
+- You will implement an event-driven architecture (EDA) with a message broker in its "center."
+
+- Streetlights will publish information about their environmental lighting to the broker.
+
+- Your application will connect to the broker and receive a stream of events from all the streetlights reporting their conditions.
+
+- Your application decides based on events when to turn the streetlight off.
+
+- Your application is not aware of how many streetlights are publishing events - it just connects to the broker and receives all events.
+
+
+# Background Context
+
+Event-driven architecture (EDA) is a design pattern built around the production, detection, and reaction to events that take place in time. In this pattern, a message broker, event publishers and subscribers are its main components for event exchange within microservices.
+
+[Message brokers](https://deploy-preview-601--asyncapi-website.netlify.app/docs/tutorials/getting-started/event-driven-architectures#message-broker) enables asynchronous communications between services so that the sending service need not wait for the receiving service’s reply. This allows interdependent services to “talk” with one another directly, even if they were written in different languages or implemented on different platforms. 
+
+Futhermore, the [Pub/sub](https://deploy-preview-601--asyncapi-website.netlify.app/docs/tutorials/getting-started/event-driven-architectures#publishersubscriber) is appealing for IoT use cases due to two key features: support for flexible coupling between publishers/subscribers and inherent support for point-to-multipoint transmission.  
+
+[MQTT](https://mqtt.org/), is a well-known protocol that is widely used in IoT applications because it was created particularly to address machine-to-machine (M2M) communication.
+
+# Installation Guide
```

Comment:
since this document is only about creating a file, I guess we do not need this section

Reply:
Hi @derberg, do you mean the installation guide section or the paragraph above it?

Reply:
the installation guide
basically to write AsyncAPI document you do not need to install anything, and in your tutorial no software is really needed

Reply:
Got it now!